R/api.R

```diff
@@ -118,33 +116,18 @@ target_table_cost_config <- function() {
 }
 
 
-endpoint_table_impact_data <- function() {
+endpoint_table_data <- function() {
   root <- schema_root()
   pkgapi::pkgapi_endpoint$new(
-    "POST", "/table/impact/data", target_table_impact_data,
+    "POST", "/table/data", target_table_data,
     pkgapi::pkgapi_input_body_json("options", "DataOptions.schema", root),
     returning = pkgapi::pkgapi_returning_json("Data.schema", root))
 }
 
 
-target_table_impact_data <- function(options) {
+target_table_data <- function(options) {
   force(options)
-  read_json(mintr_path("json/table_impact_data.json"))
-}
-
-
-endpoint_table_cost_data <- function() {
-  root <- schema_root()
-  pkgapi::pkgapi_endpoint$new(
-    "POST", "/table/cost/data", target_table_cost_data,
-    pkgapi::pkgapi_input_body_json("options", "DataOptions.schema", root),
-    returning = pkgapi::pkgapi_returning_json("Data.schema", root))
-}
-
-
-target_table_cost_data <- function(options) {
-  force(options)
-  read_json(mintr_path("json/table_cost_data.json"))
+  read_json(mintr_path("json/table_data.json"))
```

Comment:
The intervention codes here do not match those used in the prevalence endpoint - see db.R:mint_intervention for a list. We should put the canonical set of these into a schema, I think